rest/model/data/meta.go

```diff
@@ -179,6 +179,13 @@ func FormatInterface(i interface{}) string {
 			slc = append(slc, s.(string))
 		}
 		return strings.Join(slc, ",")
+	case map[string]interface{}:
+		// Required for Terraform workaround to allow users to submit raw json of feed pointer
+		// as value for metadata.  See https://github.com/terraform-providers/terraform-provider-ns1/issues/35
+		mapVal := i.(map[string]interface{})
+		feedPtr := FeedPtr{FeedID: mapVal["feed"].(string)}
```

Comment:
do we need to ensure `mapVal["feed"]` actually exists before casting here? Or is it just assumed that the "feed" key will always exist?

Reply:
It was assume, but we should definitely ensure it exists.  Rewrote the case so it will panic if a `map[string]interface{}` is passed with any key other than `feed`, as that's the only valid value for this use case.